packages/core/src/codewhispererChat/controllers/chat/messenger/messenger.ts

```diff
@@ -438,6 +438,27 @@ export class Messenger {
         )
     }
 
+    public sendPartialBashToolLog(message: string, tabID: string, triggerID: string) {
+        this.dispatcher.sendChatMessage(
+            new ChatMessage(
+                {
+                    message,
+                    messageType: 'answer-part',
+                    followUps: undefined,
+                    followUpsHeader: undefined,
+                    relatedSuggestions: undefined,
+                    triggerID,
+                    messageID: `tool-output`,
```

Comment:
Should we use the toolUseId here? I'm not sure if this will cause unexpected conflicting messages

Reply:
Will update